accounting/entries.go

```diff
@@ -401,11 +401,22 @@ func paymentReference(sequenceNumber uint64, preimage lntypes.Preimage) string {
 
 // paymentNote creates a note for payments from our node.
 // nolint: interfacer
-func paymentNote(dest *route.Vertex) string {
-	if dest == nil {
+func paymentNote(dest *route.Vertex, memo *string) string {
+	var notes []string
+
+	if memo != nil && *memo != "" {
+		notes = append(notes, fmt.Sprintf("memo: %v", *memo))
+	}
+
+	if dest != nil {
+		notes = append(notes, fmt.Sprintf("destination: %v", dest))
+	}
+
+	if len(notes) == 0 {
```

Comment:
nit: no need for this check as `strings.Join()` will just return an empty string if `notes` is empty.

Reply:
Updated in most recent squashed commit.